fitbenchmarking/fitbenchmarking
# Removes redundant fitting/misc.py

#### Description of Work

Fixes #348 

Removes `fitting/misc.py` since functions within are either defined elsewhere or can be swapped to initialise a class with attributes


#### Testing Instructions

1. Check results are the same

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
